sympy/printing/fcode.py

```diff
@@ -178,6 +172,32 @@ def _print_Piecewise(self, expr):
                                       "inline operators is not supported in "
                                       "standards earlier than Fortran95.")
 
+    def _print_Assignment(self, expr):
```

Comment:
I agree, but the code printers are already doing these kinds of hacks, and it's a bit of a slippery slope. I actually think it might be better to simply not support `sign` for Fortran standards < 95 since these fixes clutters the printers. Or maybe we should have a different printer class for writing statements and blocks thereof.